### PR TITLE
OLCF script update

### PR DESCRIPTION
**Pull request by prckent**
Updated scripts for OLCF machines. They now pull from GitHub. Same script recipe as for oxygen.

### CODE REVIEW

File: tests/test_automation/nightly_ornl_olcf_titan_gpu.job [hunk 1]
```diff
@@ -117,42 +122,47 @@ case $sys in
...
+if [ -e ${test_dir}/${machine_name}_build_gnu/bin/qmcpack ]; then
 echo "--- Found a built qmcpack executable. Will requeue."
-if [ -e $HOME/.qmcpack_test_jobs/nightly_ornl_olcf_titan_gpu.job ]; then
-# Run after 11.01pm following Tuesday. This is usually the evening after maintenance
-qsub -a `date -d "next Tuesday" +%m%d2301` $HOME/.qmcpack_test_jobs/nightly_ornl_olcf_titan_gpu.job
+cd $HOME/.qmcpack_test_jobs
+if [ -e nightly_ornl_olcf_titan_cpu.job ]; then
+# Run after 11.21pm following Tuesday. This is usually the evening after maintenance
+echo -- Qsub
+qsub -a `date -d "next Tuesday" +%m%d2321` nightly_ornl_olcf_titan_cpu.job
```
Comment (markdewing) [inline tests/test_automation/nightly_ornl_olcf_titan_gpu.job]: Should this be submitting the  *_titan_gpu.job to requeue?

Comment (prckent) [inline tests/test_automation/nightly_ornl_olcf_titan_gpu.job]: Correct. Will now find out how to update the PR.

Comment (markdewing) [inline tests/test_automation/nightly_ornl_olcf_titan_gpu.job]: Push changes to the same branch, and it should automatically update the PR.

Comment (prckent) [inline tests/test_automation/nightly_ornl_olcf_titan_gpu.job]: Fixed.

**Pull request merged**